### PR TITLE
feat: configured eslint, prettier and husky

### PR DESCRIPTION
**Pull request by bhanu0187**
## Task Link  

https://www.notion.so/js-shinobis/Setup-Eslint-Prettier-for-our-repository-7d76f440ba1a4ad79354e2803457e2be?pvs=4

## Description:-  

* Added eslint, prettier and husky pre-commit hook  
* Configured all with each other
* Also added Airbnb style-guide  

## Screenshot / Gif (If Any)  
---
---

## TODO Checklist  
- [x] PR Description included

### CODE REVIEW

File: src/pages/_app.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import '@/styles/globals.css'
+/* eslint-disable react/prop-types */
+import '../styles/globals.css';
```
Comment (abhimishra01) [inline src/pages/_app.js:2]: lets see if `/styles/global.css` work in place of this

File: src/pages/api/hello.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
 
 export default function handler(req, res) {
-  res.status(200).json({ name: 'John Doe' })
+  res.status(200).json({ name: 'John Doe' });
```
Comment (abhimishra01) [inline src/pages/api/hello.js]: do we need this api route as part of this task ?

**Review by abhimishra01 — CHANGES_REQUESTED** [review]

File: src/pages/_app.js (hunk 1 above)
Comment (bhanu0187) [reply src/pages/_app.js:2]: No, it's giving error. We'll set up absolute imports later.

File: src/pages/api/hello.js (hunk 2 above)
Comment (bhanu0187) [reply src/pages/api/hello.js]: It was generated with nextJs but I don't think we need this.

**Review by abhimishra01 — APPROVED** [review]

**Pull request merged**